charged-ieee/lib.typ

```diff
@@ -31,20 +31,28 @@
   set document(title: title, author: authors.map(author => author.name))
 
   // Set the body font.
-  set text(font: "TeX Gyre Termes", size: 10pt)
+  // As of 2024-08, the IEEE LaTeX template uses wider interword spacing
+  // - See e.g. the definition \def\@IEEEinterspaceratioM{0.35} in IEEEtran.cls
+  set text(font: "TeX Gyre Termes", size: 10pt, spacing: .35em)
 
   // Enums numbering
   set enum(numbering: "1)a)i)")
 
   // Tables & figures
   set figure(placement: top)
+  show figure.caption: set align(left)
   show figure.where(kind: table): set figure.caption(position: top)
   show figure.where(kind: table): set text(size: 8pt)
-  show figure.caption.where(kind: table): smallcaps
-  show figure.where(kind: table): set figure(numbering: "I")
-
+  show figure.where(kind: table): set figure(supplement: [TABLE], numbering: "I")
   show figure.where(kind: image): set figure(supplement: [Fig.], numbering: "1")
-  show figure.caption: set text(size: 8pt)
+
+  // Caption vs. reference:  supplement "TABLE" -> "Table", "Fig." -> "Figure"
+  set ref(supplement: it => {
+    if it.supplement == [TABLE] [Table]
+    else if it.supplement == [Fig.] [Figure]
+    else [ it.supplement ]
```

Comment:
```suggestion
    else [ #it.supplement ]
```

Reply:
Thanks! The spaces around that variable were also wrong, so I'll remove those as well